docs/pages/core_lib_doc/advantages.md

```diff
@@ -11,7 +11,7 @@ toc: false
 
 
 
-#### It can be deployed anywhere.
+#### Robust Deployment.
 
 Running/Debugging `Core-Lib` is as simple as running the `__main__` function.
```

Comment:
highlight where using the word Core-Lib or any other word needed to highlighted

Reply:
replace this "It is a standard primary container for organizing and managing any Python project" 

with "It is a standard way to managing any Python project"

Reply:
"This directory contains the unit tests for the Core-Lib" or "This directory contains the unit tests for Core-Lib"?

Reply:
maybe use the word main? "A pivotal configuration"

Reply:
"This directory manages the core data layers of the library, responsible for handling data interactions and management" mention also notion chietecture with the links as before